services/ext/service.go

```diff
@@ -259,11 +261,98 @@ func (s *Service) InitProtocol(nodeName string, identity *ecdsa.PrivateKey, appD
 	return s.messenger.InitInstallations()
 }
 
+func (s *Service) startWakuMetrics() error {
```

Comment:
Perhaps overthinking, but what will happen if I call `startWakuMetrics` twice?
I'd say it's better to
```go
if s.wakumetrics != nil {
    return
}
```